### PR TITLE
Change http method in getAssetInfo and getPoliciesDecisions from POST to GET

### PR DESCRIPTION
**Pull request by rohithdv**
Based on suggestions by @roytman, this PR has the following improvements to the taxonomy:
- [x] Change getAssetInfo POST to GET
- [x] Change getPoliciesDecisions POST to GET
- [x] Add logs in opa connector
- [x] Updated client code generation for policy manager with respect to openapi-generator 5.4.0 (Current master version was compiled for version 5.3.0 which is quite dated)

PS: Once this PR is accepted, We would need updates in wkc connector also (TODO: This PR needs to be raised in WKC Connector repo).

### CODE REVIEW

File: connectors/opa/connector.go [hunk 1]
```diff
@@ -83,6 +90,8 @@ func (r *ConnectorController) GetPoliciesDecisions(c *gin.Context) {
 		c.JSON(http.StatusInternalServerError, gin.H{"error": err.Error()})
 		return
 	}
+	r.log.Info().Msg(
```
Comment (roytman) [inline connectors/opa/connector.go:93]: All other log messages are printed with `DEBUG` verbosity level, but this one with `INFO` , why?

**Review by roytman — DISMISSED** [review]
A small logging-related comment. 
Set it to draft to prevent merge, as mentioned in the PR, we have to update the WKC connector too

**Comment by roytman** [comment]
looks like [install_openapi-generator-cli.sh](https://github.com/fybrik/fybrik/blob/master/hack/tools/install_openapi-generator-cli.sh)  tries to install the latest version. In my case it was 5.4
If you want to update the generated files you have to update install_openapi-generator-cli.sh too.
and https://github.com/fybrik/fybrik/blob/7a658a8d72c22d001d84e7433ca9289568fdaddd/site/Makefile#L6

File: connectors/opa/connector.go (hunk 1 above)
Comment (rohithdv) [reply connectors/opa/connector.go]: Fixed. Changed log level to Debug in opa connector and katalog-connector. Please check. Thanks.

**Pull request closed without merging**